src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+@import "../utils/variables";
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__foo {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+  }
+
+  &__image {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+
+  &__content {
+    padding: 8px 16px 16px;
+  }
+
+  &__title {
+    font-family: $font-family;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__product-code {
+    font-family: $font-family;
+    font-weight: 400;
+    font-size: 10px;
+    color: $secondary-color;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    margin-top: 17px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: $font-family;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__review {
+    margin-top: 1px;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-color;
+
+    &-value {
+      font-size: 16px;
+      font-family: $font-family;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+    width: 164px;
+    height: 38px;
```

Comment:
Height could be determined by the content + paddings. Setting fixed height is not optimal idea.